### ISSUE TITLE
read_service_ids_by_date AttributeError in 0.6.0

### ISSUE THREAD

**Issue by cjer** [remix/partridge#28]
* partridge version: 0.6.0
* Python version: 3.6

### Description
`read_service_ids_by_date` broke on our GTFS feed in 0.6.0. Up to 0.5.0 this worked perfectly fine.
### What I Did

``` python
import partridge as ptg
#local_zip_path - taken from here ftp://gtfs.mot.gov.il/israel-public-transportation.zip
service_ids_by_date = ptg.read_service_ids_by_date(local_zip_path)
```

Throws this error
``` python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-9-07ad157544c4> in <module>()
      1 import partridge as ptg
      2 
----> 3 service_ids_by_date = ptg.read_service_ids_by_date(local_zip_path)
      4 next_thursday = None #TODO: add this instead of hardcoding the date
      5 service_ids = service_ids_by_date[datetime.date(2018, 3, 1)]

~\Anaconda3\lib\site-packages\partridge\readers.py in read_service_ids_by_date(path)
     37     '''Find all service identifiers by date'''
     38     feed = raw_feed(path)
---> 39     return _service_ids_by_date(feed)
     40 
     41 

~\Anaconda3\lib\site-packages\partridge\readers.py in _service_ids_by_date(feed)
     63     # Only consider calendar.txt/calendar_dates.txt rows with applicable trips
     64     calendar = feed.calendar[feed.calendar.service_id.isin(service_ids)].copy()
---> 65     caldates = feed.calendar_dates[feed.calendar_dates.service_id.isin(service_ids)].copy() # noqa E501
     66 
     67     if not calendar.empty:

~\Anaconda3\lib\site-packages\pandas\core\generic.py in __getattr__(self, name)
   3612             if name in self._info_axis:
   3613                 return self[name]
-> 3614             return object.__getattribute__(self, name)
   3615 
   3616     def __setattr__(self, name, value):

AttributeError: 'DataFrame' object has no attribute 'service_id'

```

**Comment by invisiblefunnel** [comment]
Thanks for the report. I've spotted the bug. Working on a fix.

**Comment by invisiblefunnel** [comment]
This should be fixed in v0.6.1. Can you confirm?

**Comment by cjer** [comment]
Indeed. Thanks!
